tests/functional/pv/pv_services/test_no_space_left.py

```
@@ -108,31 +116,46 @@ def test_pvc_no_space_left(self, base_setup):
+        logger.info(f"Running FIO to fill PVC size: {self.pvc_size_gb}")
         self.pod_obj1.run_io(
             "fs", size=self.pvc_size_gb, io_direction="write", runtime=60
         )
-        log.info("Waiting for IO results")
+        logger.info("Waiting for IO results")
         try:
             self.pod_obj1.get_fio_results()
         except ex.CommandFailed as cf:
             if "No space left on device" not in str(cf):
                 raise
             else:
```

Ruff (0.15.15)

[warning] Unnecessary `else` after `raise` statement

Remove unnecessary `else`

(RET506)